assets/scss/_custom.scss

```diff
@@ -858,3 +858,8 @@ div.alert > em.javascript-required {
   min-height: 1.5em;
   margin: calc(max(4em, ( 8vh + 4em ) / 2)) 0 0.25em 0;
 }
+
+// Consistent spacing for yaml code
+code > span[style^="display:flex;"] {
+  display: inline !important;
+}
```

Comment:
Please add the ending newline here, thanks!

Reply:
done!